extensions.json

```diff
@@ -276,6 +280,10 @@
       "version": "1.4.11",
       "repository": "https://github.com/HookyQR/VSCodeBeautify"
     },
+    {
+      "id": "hridoy.rails-snippets",
+      "repository": "https://github.com/hridoy/rails-vscode"
```

Comment:
Oops, there is no code or `package.json` file in https://github.com/hridoy/rails-vscode ? How does this work?

Reply:
Ah, sorry I didn't check that closely. I just used the repository link from the marketplace since it was MIT licensed, I thought the code would be there. Is it good if we could remove that extension for now and proceed with the other two? I'll check this one out with the author and create a new PR for it if possible. I use VSCodium and the recent update only has extension from open-vsx, so trying to get some of the extensions I used over to open-vsx. Thanks :)

Reply:
Sure! No problem, I'll merge the other two, and we can see about this one later. 🙂

Thanks for testing the VSCodium/OpenVSX integration, and especially for adding extensions you're missing! That's super helpful 💯